content_schemas/formats/shared/definitions/content_id_alias.jsonnet

```diff
@@ -1,5 +1,16 @@
 {
   content_id_alias: {
+    description: "Human-readable alias for a Content ID, used when embedding content. Should only be supplied when updating Content Blocks.",
     type: "string",
   },
+  content_id_alias_optional: {
```

Comment:
I think I follow this now, so if we were going to make it required, we'd have something like this?

```
content_id_alias_required: {
  "$ref": "#/definitions/content_id_alias",
}
```

Reply:
Oh wait, I see, all the magic is in the `Format` class - that makes sense 👍